CITATION.cff

```diff
@@ -0,0 +1,87 @@
+cff-version: 1.2.0
+title: >-
+  DAISY: A Data Information System for accountability
+  under the General Data Protection Regulation
+message: >-
+  If you use this software, please cite the paper
```

Comment:
Maybe the `message` could be written as: `If you use this software, please cite it as below.` (i.e., not mention `paper`)

Reply:
How does this sentence work? When I'm using the software, where and how do I cite it? Shouldn't it be, "Please the below citation when referencing our work in any published materials" or something?

Reply:
I like @Trefex 's suggestion more. People will hardly cite software if they just use it. I think this was in the template I have started with. 

The message is not being displayed anyway at the  moment - there is probably default GitHub message:
![Screenshot from 2022-02-04 14-45-12](https://user-images.githubusercontent.com/51602017/152542184-ba19195e-1245-4463-a532-da05ea54270a.png)

Reply:
fixed in [83fb454](https://github.com/elixir-luxembourg/daisy/pull/319/commits/83fb4541a562df715769da0e77c3d100965b835f)